node/src/main/scala/coop/rchain/node/api/ReplGrpcService.scala

```diff
@@ -67,7 +70,7 @@ private[api] class ReplGrpcService(runtime: Runtime[Task], worker: Scheduler)
                   .map(_.toString())
                   .mkString("Errors received during evaluation:\n", "\n", "\n")
             s"Deployment cost: $cost\n" +
-              s"${errorStr}Storage Contents:\n ${StoragePrinter.prettyPrint(runtime.space)}"
+              s"${errorStr}Storage Contents:\n$prettyStorage"
 
```

Comment:
why an empty line?

Reply:
Not sure what you mean?
Before the first term was offset by one space while all following terms where not.

Reply:
just a formatting question. Why keep the empty lines, line 74 & line 75?

Reply:
Oh, I see. Just removed them.